aws/ecs_cluster_test.go

```diff
@@ -0,0 +1,130 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test that we can succesfully list ECS clusters by manually creating a cluster and then using the list function to find it.
+func TestCanCreateAndListEcsCluster(t *testing.T) {
+	t.Parallel()
+
+	region := "eu-west-1"
+	awsSession, err := session.NewSession(&awsgo.Config{
+		Region: awsgo.String(region),
+	})
+	if err != nil {
+		assert.Fail(t, errors.WithStackTrace(err).Error())
+	}
```

Comment:
NIT: the slightly cleaner way to do this:

```suggestion
	require.NoError(t, err)
```